common/app/services/newsletters/NewsletterApi.scala

```diff
@@ -133,4 +111,26 @@ case class NewsletterApi(wsClient: WSClient)(implicit executionContext: Executio
     }
   }
 
+  private def getBody(path: String): Future[JsValue] = {
```

Comment:
I think this change just moves code around a bit - let me know if I've missed something!

Reply:
Yes, second commit is scalafmt, it reordered stuff. It was breaking in CI before.